server/db/models/product.js

```diff
@@ -0,0 +1,34 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Product = db.define('product', {
+  title: {
+    type: Sequelize.STRING,
+    unique: true,
+    allowNull: false
+  },
+  description: {
+    type: Sequelize.TEXT
+  },
+  price: {
+    type: Sequelize.DECIMAL(10, 2),
+    allowNull: false,
+    validate: {
+      min: 0
+    }
+  },
+  quantity: {
+    type: Sequelize.INTEGER,
+    allowNull: false,
+    validate: {
+      min: 0
+    }
+  },
+  imageUrl: {
+    type: Sequelize.STRING,
+    defaultValue:
+      'https://i.pinimg.com/originals/2d/94/33/2d94334b737efb5d3a5ef32aef9daefc.jpg'
```

Comment:
Consider making the default value a relative link to an image (make sure to downscale it!) you put into your public folder, e.g. `images/example_product.jpg`. It shouldn't be a massive picture, but something on the order of tens or low hundreds of Kb can be added to source code / version control without major issue.

A dependency on a 3rd-party service like this for the sample image is ok if you are very confident the service is A) allowed to be used for hosting / hotlinking, B) not going away any time soon, and C) has stable URLs. Or, if the linked image is non-critical.